README.md

```diff
@@ -438,7 +438,7 @@ Table of Contents
 
 ## Code Search and Browsing
 
-  * [codota.com](https://www.codota.com/) — Codota helps developers create better software, faster by providing insights learned from all the code in the world. Plugin available.
+  * [tabnine.com(formerly codota.com)](https://www.tabnine.com) — Codota helps developers create better software, faster by providing insights learned from all the code in the world. Plugin available.
```

Comment:
Can probably just drop the “(formally …)” bit and also update the name in tbe description area